ticketutil/bugzilla.py

```diff
@@ -231,6 +234,45 @@ def add_comment(self, comment, **kwargs):
             logging.error("Error adding comment to ticket")
             logging.error(e.args[0])
 
+    def add_attachment(self, data, file_name, summary, **kwargs):
```

Comment:
Change the order of the named parameters to:

file_name, data, summary

Also, change the order of the :param tags in lines 239, 240, 241 to stay consistent.